unix.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:  1.12
 name:           unix
-version:        2.8.0.0
+version:        2.9.0.0
```

Comment:
Please keep version 2.8.0.0, the latest release was 2.7.

Reply:
The idea was to bump beyond 2.8 on purpose, so that another major point release can be maintained in between that does not include AFPP.

Reply:
Please stick to 2.8, we'll make amends if needed later.